logic/validator_logic.py

```diff
@@ -38,9 +39,17 @@ def extract_validator_from_snapshot(info_json):
         uptime = None
         name = address
     bls_keys = info_json['keys-at-auction']
-    slots = len(bls_keys)
+    bls_keys_per_shard = defaultdict(int)
+    allowed_keys: any
+    for bls_key in bls_keys:
+        shard = int(bls_key, 16) % 4
+        if bls_keys_per_shard[shard] < 13:
```

Comment:
Can you make `13` a constant in config.py

Reply:
Sure, done.